src/Shared/LayoutRenderers/AspNetRequestServerVariableLayoutRenderer.cs

```diff
@@ -0,0 +1,71 @@
+using System.Text;
+using NLog.LayoutRenderers;
+using NLog.Web.Internal;
+#if !ASP_NET_CORE
+using System.Web;
+#else
+using Microsoft.AspNetCore.Http.Features;
+#endif
+
+namespace NLog.Web.LayoutRenderers
+{
+    /// <summary>
+    /// ASP.NET Server Variable.
+    /// </summary>
+    /// <remarks>
+    /// Use this layout renderer to insert the value of the specified Server Variable
+    /// </remarks>
+    /// <example>
+    /// <para>Example usage of ${aspnet-request-servervariable}:</para>
+    /// <code lang="NLog Layout Renderer">
+    /// ${aspnet-request-servervariable:ServerVariable=v}
+    /// </code>
+    /// </example>
+    [LayoutRenderer("aspnet-request-servervariable")]
+    public class AspNetRequestServerVariableLayoutRenderer : AspNetLayoutRendererBase
+    {
+#if !ASP_NET_CORE || ASP_NET_CORE3
```

Comment:
Can `#if !ASP_NET_CORE || ASP_NET_CORE3` be moved outside, so it is the entire class.

Reply:
I missed that, sorry, it was that way in the original AspNetRequestValueLayoutRender class.  Completed.